### PR TITLE
supported devices list included within yalu

### PR DESCRIPTION
**Pull request by gzimbric**
- added about page to yalu app which includes supported devices
- credits for yalu developers moved to separate page

### CODE REVIEW

File: yalu102.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -174,7 +177,7 @@
 				TargetAttributes = {
 					EA9900E21E1E9F060056FEBD = {
 						CreatedOnToolsVersion = 8.2.1;
-						DevelopmentTeam = CGTX3WH3ZS;
```
Comment (bobbysn0w) [inline yalu102.xcodeproj/project.pbxproj]: Ya messed up a bit there buddy

File: yalu102.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -369,14 +373,14 @@
 				ARCHS = armv7;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_OBJC_ARC = NO;
-				DEVELOPMENT_TEAM = CGTX3WH3ZS;
```
Comment (bobbysn0w) [inline yalu102.xcodeproj/project.pbxproj]: And again

File: yalu102.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -369,14 +373,14 @@
...
+				DEVELOPMENT_TEAM = H65Y924PHT;
 				INFOPLIST_FILE = yalu102/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				LIBRARY_SEARCH_PATHS = (
 					"$(inherited)",
 					"$(PROJECT_DIR)/yalu102",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = kim.cracksby.yalu102;
```
Comment (bobbysn0w) [inline yalu102.xcodeproj/project.pbxproj]: And again

File: yalu102.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -388,14 +392,14 @@
 				ARCHS = armv7;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_OBJC_ARC = NO;
-				DEVELOPMENT_TEAM = CGTX3WH3ZS;
```
Comment (bobbysn0w) [inline yalu102.xcodeproj/project.pbxproj]: uh huh

File: yalu102.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -388,14 +392,14 @@
...
+				DEVELOPMENT_TEAM = H65Y924PHT;
 				INFOPLIST_FILE = yalu102/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				LIBRARY_SEARCH_PATHS = (
 					"$(inherited)",
 					"$(PROJECT_DIR)/yalu102",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = kim.cracksby.yalu102;
```
Comment (bobbysn0w) [inline yalu102.xcodeproj/project.pbxproj]: and again

**Comment by bobbysn0w** [comment]
I don't see how this is needed because the IPA would be downloaded from the official website so...

File: yalu102.xcodeproj/project.pbxproj (hunk 5 above)
Comment (gzimbric) [inline yalu102.xcodeproj/project.pbxproj]: ....... thx alot

**Pull request closed without merging**